website/content/partials/telemetry-metrics/vault/client/billing_period/activity.mdx

```diff
@@ -0,0 +1,7 @@
+### vault.client.billing_period.activity ((#vault.client.billing_period.activity)) <EnterpriseAlert inline="true" />
+
+Metric type | Value   | Description
+----------- | ------- | -----------
+gauge       | clients | The number of distinct clients across all namespaces seen since from the start of the billing period to the current moment.
```

Comment:
```suggestion
gauge       | clients | The number of distinct clients across all namespaces seen since from the start of the current billing period to the current moment.
```